scripts/tests/create_test_clusters.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Change to the directory where the script is located
+cd "$(dirname "$0")"
+
+# create SKR cluster
+if k3d cluster list | grep -q "skr"; then
```

Comment:
this command would also get a cluster named `skr-local` for example, so I think this should be `k3d cluster list | grep "^skr\s"` to do an exact match of the name "skr"